cmd/main.go

```diff
@@ -47,7 +54,10 @@ func init() {
 	strfmt.MarshalFormat = strfmt.ISO8601LocalTime
 }
 
-const deploymet_type_k8s = "k8s"
+const (
+	deploymet_type_k8s  = "k8s"
+	leaderConfigMapName = "assisted-service-leader-election-helper"
```

Comment:
if it's a const why do you define it here? it can be set as default config or just have inside the new function.

Reply:
didn't want to provide it as config. i can provide it as string without having consts. Thought it will be easier to see. In theory u can have many leaders just each of them need to have it's own config map.
I think it can be useful when we may think about splitting workload

Reply:
Do you see any use case for it in our service?

Reply:
But maybe it's fine if it's a generic package.